src/main/java/org/app/TransactionsController.java

```diff
@@ -23,7 +22,7 @@
 @RestController
 class TransactionsController {
     @GetMapping("transactions")
-    public ConcurrentHashMap<Integer,Transaction> getTransactions() {
+    public LinkedList<Transaction> getTransactions() {
```

Comment:
Trata siempre de utilizar interfaces en vez de implementaciones

```suggestion
    public List<Transaction> getTransactions() {
```

Reply:
Cállate tú no sabes.